### DISCUSSION TITLE
Diedrich DR-25 annotations

### DISCUSSION THREAD

**Discussion by WhitneyCoffee** [artisan-roaster-scope/artisan#904] [General]
Hi,
![IMG_5490 2](https://user-images.githubusercontent.com/107888521/174672597-aa54f264-e80d-4afe-9791-3dffc5f71620.jpg)

I have had a hard time hooking up my roaster. Things are logging now but I can seem to get my my events to show up on my curve
charge, dry end FC etc. in config-events-config have "show on BT' & 'annotations' checked. but they dont show up on curve when i press the buttons. Artisan 2.6.0

Any help would be greatly appreciated.

**Comment by poundy** [comment]
time to consult the manual ;-) 
https://artisan-scope.org/docs/background/
